distribution/src/conf/log4j2.properties

```diff
@@ -457,7 +453,7 @@ logger.StAXDialectDetector.level = ERROR
 #uncomment the last line to add syslog appender to the root loggers
 rootLogger.level = INFO
 rootLogger.appenderRef.CARBON_CONSOLE.ref = CARBON_CONSOLE
-rootLogger.appenderRef.CARBON_LOGFILE.ref = CARBON_LOGFILE
+#rootLogger.appenderRef.CARBON_LOGFILE.ref = CARBON_LOGFILE
```

Comment:
Shall we remove this commented out line as well

Reply:
We have decided to keep it since if someone has decided to append to the CARBON_LOGFILE instead of CARBON_CONSOLE he can just uncomment this and comment the console part